### DISCUSSION TITLE
Can you make a short informative video?

### DISCUSSION THREAD

**Discussion by kellegogg31** [sk-zk/Extractor#14] [Q&A]
![image](https://github.com/user-attachments/assets/3ee6aa2f-d254-4aff-b560-edcb7819ec33)
hello, I have downloaded your latest version application, my file.scs unfortunately I can't open it at all. i'm getting a damaged alert i feel like I've done something wrong.
sorry for my bad English

**Comment by sk-zk** [comment]
My extractor is a command line tool, just like the official one, so you run it from a terminal:

1) Open the Command Prompt from the start menu or with Win+R `cmd`.
2) Use the `cd` command  to go to the folder where the .exe is located, e.g.:
```batch
C:\Users\Foo>cd "D:\Downloads\extractor"
```
If the folder displayed on the left is on a different partition than the one you just typed in, you will need to follow it up by just typing in the letter of the partition:
```batch
C:\Users\Foo>D:
```

3) Call the extractor with the path of the .scs file:
```batch
D:\Downloads\extractor>extractor "G:\New folder\Pro-bd def.scs"
```

You can also drag and drop your .scs file onto the extractor, but if it complains about a missing top level directory, you will need to call it directly from the command line to supply additional options.

**Reply by kellegogg31** [reply]
![image](https://github.com/user-attachments/assets/cb5f08f3-492a-4a1f-81ac-ad54b049ac43)
![image](https://github.com/user-attachments/assets/937df26a-80c7-4ba4-bf1c-62e430acce23)
![Adsız](https://github.com/user-attachments/assets/4e6df3b2-40d0-4ddb-9b77-b0db8c0ee296)
-a 
i did what you said, the program is working, but unfortunately my file is not being opened.
-a i'm using the command damaged says wrong

**Reply by sk-zk** [reply]
It's not really clear what you're trying to do. If you want to extract that specific file, and both the extractor and the file are located in the working directory, the command is `extractor renault.scs`. If you want to extract all .scs files in the working directory, use `extractor -a`. If you want to extract all .scs files in a specific folder, use `extractor -a "path\to\the\folder"`.

**Reply by jonesy-91** [reply]
Hello, I'm new to using this program, is there anyone that could help me open a few locked/encrypted mods, purchased some mods, was told they wasn't locked but of course they are :(

**Reply by cricardo10d-ui** [reply]
> It's not really clear what you're trying to do. If you want to extract that specific file, and both the extractor and the file are located in the working directory, the command is `extractor renault.scs`. If you want to extract all .scs files in the working directory, use `extractor -a`. If you want to extract all .scs files in a specific folder, use `extractor -a "path\to\the\folder"`.

Hello u can help me, i try to use the tool, but i can't get it, i'm so stressed

**Reply by sk-zk** [reply]
> Hello u can help me, i try to use the tool, but i can't get it, i'm so stressed

Take a look at the sample calls at the bottom of the readme to get started. If you've never used the command line before, you can find a lot of introductory material via Google.
